bravebookmarks/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id" :              "org.albert.extension.bravebookmarks",
+    "name" :            "Brave bookmarks",
+    "version" :         "1.0alpha",
+    "platform" :        "Linux",
+    "group" :           "Extensions",
+    "author" :          "Manuel Schneider",
```

Comment:
Is that the author though?

Reply:
I can't in all honesty claim authorship for this code, it's literally a Find and Replace job on the Chrome plugin! :smile: Leave it as Manuel.

In the absence of an Albert Contributor Covenant: I'm happy to contribute this to the Albert project and I don't claim any copyright over the work. I assign ownership to the Albert project or to @ManuelSchneid3r , whichever is better for you.